README.md

```diff
@@ -1,10 +1,12 @@
 # tbfighters.org
-Source Code of the tbfighters.org website, written in vanilla HTML, CSS and JavaScript to keep a light footprint.
+Source Code of the [tbfighters.org](https://tbfighters.org) website, written in vanilla HTML, CSS and JavaScript to keep a light footprint.
 
 ## Contributing
 Contributions are gladly appreciated!
 Feel free to fork and create a pull request with the changes you would like to propose.
 However, please note that we may not accept all pull requests as we want to ensure a certain level of factuality and quality on the site.
+You can view our Contributor Guide [here](https://docs.google.com/document/d/1fM523NeS6SUvp-dweWrALmPBJrtPF0QCcNH9zqvjILY), which goes into what we believe to be best practices when contributign to this repository!
+
 By contributing to this project, you license all your code and other text materials under the MIT license, and all media content must either be in the public domain or licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).
 
 View the [contributing guide](CONTRIBUTING.md) for more info.
```

Comment:
Hm... Just rebased and realized this got added too. Seems we have 2 contributor guides now, should I move the google drive link into this file instead?

Reply:
I would say that the google doc is like how to use github and stuff so you could say its that. The other one is more the technical details on how to use the tools that we've made